static-assets/components/cstudio-forms/forms-engine.js

```diff
@@ -728,6 +732,27 @@ var CStudioForms = CStudioForms || function() {
     sendMessage({ type: FORM_REQUEST, key });
   }
 
+  function createDialog() {
+    var dialogEl = document.getElementById('saveDraftWar');
+    if (!dialogEl) {
+      var dialog = new YAHOO.widget.SimpleDialog('saveDraftWar',
```

Comment:
Let's use a more subtle notification - something less disruptive. Use noty lib to show a toast-like save notification.